angularjs/ListView/Getting-Started.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting started | ListView | JavaScript | Syncfusion
-description: Getting started with listview widget in angularjs
+title: Getting Started with AngularJS ListView control | Syncfusion
+description:Learn here all about Getting started with Syncfusion Essential AngularJS ListView control, its elements, and more.
```

Comment:
Update as below
description: Learn here all about Getting started with Syncfusion Essential AngularJS ListView control, its elements, and more.
![image](https://github.com/syncfusion-content/angularjs-docs/assets/95272441/eb207638-4d4b-4d1f-baed-de188f8a98a9)